src/components/sidebar/sidebar.stories.mdx

```diff
@@ -42,7 +43,7 @@ Useful to perform an action in context without navigating the user to a separate
 
 Useful if the user might need to refer back to the page behind the sidebar. For example, the user could select from a list of clients on the page, with detailed information on the client loading in the sidebar.
 
-Several pre-set widths are available, and the height is always 100%. It’s best to avoid sidebars that are taller than the user’s viewport height, as a scroll would hide some information. Typical user viewport heights can be as little as 650 pixels.
+The height of the sidebar is always 100%. It’s best to avoid sidebars that are taller than the user’s viewport height, as a scroll would hide some information. Typical user viewport heights can be as little as 650 pixels.
```

Comment:
I'm not sure why you've removed the part of the docs about several pre-set widths being available? As far as I can tell from the code, there is still a `size` prop which takes one of the predefined size strings in SIDEBAR_SIZES. I don't see why adding padding props should change this, and no notes in your PR about anything to do with this?